hilti/toolchain/include/ast/node.h

```diff
@@ -643,6 +643,13 @@ class Node {
      */
     void replaceChild(ASTContext* ctx, Node* old, Node* new_);
 
+    /**
+     * Removes the node from its parent. The node will remain valid and can be
+     * re-inserted into the AST elsewhere later. Does nothing if the node has
+     * no parent. If the node has no parent, this is a no-op.
```

Comment:
the two sentences say essentially the same thing


```suggestion
     * no parent.
```